aws_adfs/login.py

```diff
@@ -156,7 +163,7 @@ def login(
         RoleArn=config.role_arn,
         PrincipalArn=principal_arn,
         SAMLAssertion=assertion,
-        DurationSeconds=3600,
+        DurationSeconds=config.session_duration,
```

Comment:
FYI If you run it a second time (with the session duration in the config) it will crash:
``` bash
botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid type for parameter DurationSeconds, value: 43200, type: <class 'str'>, valid types: <class 'int'>
```

Verified on my version:
https://github.com/KyleJamesWalker/aws-adfs/blob/session-duration/aws_adfs/login.py#L166